autogluon/utils/tabular/ml/models/abstract/abstract_model.py

```diff
@@ -244,7 +248,7 @@ def compute_feature_importance(self, X, y, features_to_use=None, preprocess=True
         banned_features = [feature for feature, importance in feature_importance_quick_dict.items() if importance == 0 and feature in features]
         features = [feature for feature in features if feature not in banned_features]
 
-        permutation_importance_dict = self.compute_permutation_importance(X=X, y=y, features=features, preprocess=False)
+        permutation_importance_dict = self.compute_permutation_importance(X=X, y=y, features=features, preprocess=False, silent=silent)
 
         feature_importances = pd.Series(permutation_importance_dict)
```

Comment:
Should the pd.Series be printed at the end of this function when `silent=False`?

Reply:
silent is to print progress of task since this can take a very long time unlike leaderboard. I'm unsure if we really want to print the series in this case, since it could be very large if 1000+ features.

Reply:
Won't pandas already handle printing of large Series for us?  I don't think it's a big deal either way.